config/logicsoft/ZDB5100.xml

```diff
@@ -0,0 +1,498 @@
+      <Item label="Same as 1" value="3"/>
+      <Item label="Same as 2" value="4"/>
+      <Item label="The LED indication follows the status of the internal dimmer, ON or OFF" value="5"/>
+      <Item label="The LED indication follows the status of the internal dimmer with inverted functionality, ON or OFF" value="6"/>
+      <Item label="The LED indicator is ON for 5 seconds when the pushbutton is activated" value="7"/>
+    </Value>
+    <Value genre="config" index="45" instance="1" label="Colour commands for pushbutton 4 LED indication" max="1" min="0" size="1" type="list" value="1">
+      <Help>This parameter specifies how received Binary Switch Setcommands are handled.</Help>
+      <Item label="Direct control - the LED indication shows the received colour immediately, until the pushbutton is activated" value="0"/>
+      <Item label="Colour command sets the colour for OFF indication" value="1"/>
+      <Item label="Colour command sets the colour for ON indication" value="2"/>
+    </Value>
+    <Value genre="config" index="46" instance="1" label="ON indication RGB-saturation for pushbutton 4" size="4" type="int" value="32512">
+      <Help>        
+        Byte 1: Colour saturation, red
+            0-255 Specifies the saturation for the red LEDs (Default is 0).
+        Byte 2: Colour saturation, green
+            0–255 Specifies the saturation for the green LEDs (Default is 0).
+        Byte 3: Colour saturation, blue
+            0-255 Specifies the saturation for the blue LEDs (Default is 127).
+        Byte 4: LED control
+            0-255 LED indicator control - see the application note about this parameter (Default is 0).
+        Default: 32512
+			</Help>
+    </Value>
+    <Value genre="config" index="47" instance="1" label="OFF indication RGB-saturation for pushbutton 4" size="4" type="int" value="50065152">
+      <Help>        
+        Byte 1: Colour saturation, red
+            0-255 Specifies the saturation for the red LEDs (Default is 47).
+        Byte 2: Colour saturation, green
+            0–255 Specifies the saturation for the green LEDs (Default is 47).
+        Byte 3: Colour saturation, blue
+            0-255 Specifies the saturation for the blue LEDs (Default is 47).
+        Byte 4: LED control
+            0-255 LED indicator control - see the application note about this parameter (Default is 0).
+        Default: 50065152
+			</Help>
+    </Value>
+  </CommandClass>
+  <!-- COMMAND_CLASS_SWITCH_ALL SwitchAllCmd_Get is not supported -->
+  <CommandClass id="39">
+    <Compatibility>
+      <GetSupported>false</GetSupported>
+    </Compatibility>
+  </CommandClass>
+  <!-- MultiInstance Command Class. Note: If you do not need Dedicated Relay Control instance, change below mapping to "endpoints", then rejoin the node to your network. -->
+  <CommandClass id="96">
+    <Compatibility>
+      <MapRootToEndpoint>true</MapRootToEndpoint>
```

Comment:
Also - Are you sure?

Reply:
No I am not. I looked at the ZHC5010 which is a similar device from same manufacturer. This will be removed in next commit.